src/templates/department.js

```diff
@@ -134,39 +136,51 @@ function DepartmentTemplate({ data }) {
   );
 }
 
+DepartmentTemplate.propTypes = {
+  data: PropTypes.object
+};
+
 export default DepartmentTemplate;
 
-function processDepartmentInfo({ node, staffDirectorySlug }) {
-  const { relationships, field_email, field_fax_number, title } = node;
+function processDepartmentInfo ({ node, staffDirectorySlug }) {
+  const { relationships, field_email: fieldEmail, field_fax_number: fieldFaxNumber, title } = node;
 
   const leadership = relationships.field_department_head
     ? {
         icon: 'person_outline',
         content: (
-          <React.Fragment>
+          <>
```

Comment:
Most of the code changes are linting changes. The important update starts here @erinesullivan.